src/modules/cloudflare/captcha/captcha.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable, InternalServerErrorException } from '@nestjs/common';
+
+import { TokenSchemaDto } from './dtos/token.dto';
+
+@Injectable()
+export class CaptchaService {
+  async captchaVerification({ token }: TokenSchemaDto): Promise<any> {
+    try {
+      const formData = new FormData();
+      formData.append('secret', process.env.CAPTCHA_SECRET_KEY);
+      formData.append('response', token);
```

Comment:
could you moving please to configService ?

const secretKey = this.configService.get<string>('CAPTCHA_SECRET_KEY');
const hostVerificationUrl = this.configService.get<string>('CAPTCHA_HOST_VERIFICATION');

if (!secretKey || !hostVerificationUrl) {
  throw new Error('CAPTCHA environment variables are not configured properly.');
}

Reply:
https://docs.nestjs.com/techniques/configuration